docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1327,6 +1327,207 @@ The ACS will be used for V&V testing. It is defined as:
+the year, they will be duplicated and receive multiple forms. 
+
+
+The rate of the the types of forms are below. This data is 
+from a review of 2016 tax data by [Lim_2019]_ 
+
+.. list-table:: Percent W2 versus 1099  
+  :widths: 10 10 
+  :header-rows: 1
+
+  * - Form Type 
+    - Percent Receiving 
+  * - W2 
+    - 94.65% 
+  * - 1099 
+    - 5.35%
+
+
+**Data Errors/Noise** 
+In the future, we will add a noise function designed to replicate 
+missed or incorrect data. This includes incorrect name spelling, 
+addresses, age, and person swaps. 
+
+Some additional tax specific errors include: SSN randomly being filled 
+for those who are missing one, mailing address being different than the 
+home address, and issues with employer names, addresses, or IDs. 
+
+These are not currently modeled. 
+
+**Limitations and Possible Future Adds** 
+
+#. Sampling on a single time step is not representative of how tax documents are compiled. 
+#. Errors are made in W2s and 1099s by companies frequently, due to employees moving or changing information without communicating changes. These W2s can be reissued which leads to duplicates, or employees might not adjust them leading to different information between W2/1099s and 1040 forms. This is not currently modeled. 
+#. 1099 forms are often used by self-employed people or those with small businesses. These can contain errors related in employer information. 
+#. There are some employed people who do not receive a W2 or 1099, often for "under the table" work. This phenomenon might be easiest to include in the simulation as these individuals would not have a listed employer despite having an income. I chose to have all those that have an employer listed receive a W2/1099. 
+#. Many workers might have multiple jobs simultaneously and receive multiple forms. This is not included in the current model. 
+#. Elderly people can still have to file taxes based on social security payments, but would likely not have an employer in our model. 
+
+
+1040 Form
+'''''''''
+
+**When to Sample** 
+
+- Sample compiled on the time step containing April 15th of each year 
+- Sample on a single time step for now 
+
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20 20 
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+    - Notes 
+  * - First name
+    - 
+  * - Middle initial 
+    - 
+  * - Last name
+    - 
+  * - Age
+    -  
+  * - DOB
+    -  
+  * - Mailing Address
+    -  
+  * - Social Security Number 
+    -
+  * - Income 
+    -
+  * - Employer ID
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Employer Address 
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Employer Zip Code 
+    - Can have multiple rows if simulant has multiple jobs in the prior year (multiple W2/1099 forms)  
+  * - Joint Filer or Dependent 
```

Comment:
if only one form, how would you like to choose who submits the form?

Reply:
actually --- i have only ever filed single, and don't know what filing jointly looks like, so this might be naive, but --- if filing jointly, do all of those identification columns get duplicated? like person_1 name, person_2 name, person_1 employer_id, person_2 employer_id, etc.?

Reply:
Honestly, I have never filed jointly either! I have expanded the table a bit to try and make this clearer. I believe yes, for the joint filing you would just include all of the information for the spouse - name, employer info, etc. I separated our dependents to try and make this more clear. 

Only one simulant would submit the form, the other is just included as a joint filer. So the reference person would submit the form, and the spouse would not submit a form. I added this as well.